src/lua/init.c

```diff
@@ -229,6 +230,7 @@ static const char *lua_modules[] = {
 	"utils.avl", utils_avl_lua,
 	"utils.bufread", utils_bufread_lua,
 	"utils.symtab", utils_symtab_lua,
+	"utils.evread", utils_evread_lua,
```

Comment:
Why not alphabetically sorted?

Reply:
Because the symtab module is used by evread, hence it should go first. In other places order has no such meaning.

Reply:
Thanks!